README.md

```diff
@@ -22,18 +18,24 @@ Thus, this program can be utilised to draw paths for automation purposes which w
 
 
 On the other hand, the car will also be able to be controlled via a joystick and save the path that was driven.
```

Comment:
This seems like something that should go into the what section rather than the why section